src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.c

```diff
@@ -41,9 +41,9 @@ void BgInibsMovebg_Init(Actor* thisx, PlayState* play) {
     Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
     DynaPolyActor_Init(&this->dyna, 1);
 
-    this->unk_15C = D_80B96560[BGINIBSMOVEBG_GET_F(thisx)];
-    this->unk_160 = D_80B96568[BGINIBSMOVEBG_GET_F(thisx)];
-    this->unk_164 = D_80B96570[BGINIBSMOVEBG_GET_F(thisx)];
+    this->opaDList = sOpaDLists[BG_INIBS_MOVEBG_GET_SIZE(thisx)];
```

Comment:
Small nit, but since you call the assets `modes` I think it would make more sense to call the macro `GET_MODE` instead of `GET_SIZE`, though not a big thing.